vue/sbc-common-components/src/models/PayData.ts

```diff
@@ -0,0 +1,15 @@
+/* response from Pay API "fees" endpoint */
+export interface PayData {
+  filingFees: number;
```

Comment:
I Googled a bit and most example use semi-colons. But I think commas, or even nothing, are also acceptable. Do you have a preference?

Reply:
OK, I removed the semi-colons from the interface properties in the latest commit.

> > > > no changes in Tests?
> > > 
> > > 
> > > I ran the tests and they still fail 😛
> > > Seriously though, it doesn't look like my changes caused any new unit test errors 🤷‍♂
> > 
> > 
> > :-) really ..I thought the tests were all running ..will be great to address the tests before it becomes unmanageable.. if time permits
> 
> Agreed. How about creating a new ticket for that, since it will probably take a day?

PS I updated the jest config and solved the "invalid token" / transpile error in SbcHeader in the latest commit.